h1st/model/k1st/collaborator_modeler.py

```diff
@@ -0,0 +1,90 @@
+import logging
+
+import pandas as pd
+
+from h1st.model.wrapper.multi_modeler import MultiModeler
+from h1st.model.oracle.ensemble import MajorityVotingEnsemble
+from h1st.model.rule_based_modeler import RuleBasedModeler
+from h1st.model.k1st.collaborator import kCollaboratorModel
+
+
+class kCollaboratorModeler(MultiModeler):
+
+    model_class = kCollaboratorModel
+
+    def __init__(self):
+        super().__init__()
+
+    def build_model(self, prepared_data: dict,
+                    modelers: list['h1st.MLModeler']=[],
+                    ensemble_modeler: 'h1st.Modeler' = RuleBasedModeler(MajorityVotingEnsemble),
+                    models: list['h1st.PredictiveModel']=None,
+                    inject_x_in_ensembler: bool=False) -> 'BaseOracle':
```

Comment:
this one mean build_model return a string value?

Reply:
No it returns a kCollaboratorModel. Forgot to fix type annotation.  Corrected now, but also strings instead of importing the actual class are accept methods of type annotating.